src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
```

Comment:
Creating a square, you should avoid passing expressions directly in the constructor. It's better to create a separate variable for the side length and pass it on for better code readability.